src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -174,11 +201,25 @@ def generate_WDB_2_default_config(self, filename):
     convertRequestToRequestWithCpiSignature(cpi_private_key_d, cpi_id_d,
                                             cpi_name_d, device_d)
 
-    # Create the actual config.
+    cpi_database_config = {
+        'hostName': 'localhost',
+        'port': 8003,
+        'cpis': [{
+            'cpiId': cpi_id_b,
+            'status': 'ACTIVE',
+            'publicKeyIdentifierFile': os.path.join('testcases', 'testdata', 'cpi_db', 'CPI_Database-Public.csv')},
+            {
+                'cpiId': cpi_id_d,
```

Comment:
Please use consistent indentation for these two CPI dicts so that it's clear they're in the same list.

Reply:
Fixed as per suggestion.